apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/service/OnboardingServiceDefault.java

```diff
@@ -771,17 +772,12 @@ private Uni<String> checkOnboardingIdFormat(String onboardingId) {
 
     private Uni<Onboarding> setInstitutionType(Onboarding onboarding) {
         return institutionRegistryProxyApi.findInstitutionUsingGET(onboarding.getInstitution().getTaxCode(), null, null)
```

Comment:
what happen when proxy return null value? you could add InstitutionType.PA as default....